app/controllers/demo/school_search_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Demo
+  class SchoolSearchController < ApplicationController
+    def index
+
+    end
```

Comment:
We can make it shorter - maybe even remove it altogether. `def index; end`?

Reply:
We could just remove it too.

Reply:
burned it with rubocop - burn baby burn